subquery/docker-compose.yml

```diff
@@ -69,7 +69,7 @@ services:
       DB_DATABASE: postgres
       DB_HOST: postgres
       DB_PORT: 5432
-      NODE_ENV: production
+      # NODE_ENV: production
```

Comment:
Do you need to keep this line commented?

Reply:
Hi Alex NODE_ENV: production is not required to be present at the moment, this will disable the playground and debug logs from the graphql engine
At the moment we need playground running for testing and so. We can uncomment the Flag later